indico/modules/events/timetable/clone.py

```diff
@@ -33,8 +33,15 @@ def is_default(self):
     def is_visible(self):
         return self.old_event.type_ in {EventType.meeting, EventType.conference}
 
-    def has_conflicts(self, target_event):
-        return self._has_content(target_event) or self.old_event.duration > target_event.duration
+    def get_conflicts(self, target_event):
+        conflicts = []
+        if self._has_content(target_event):
+            conflicts.append(_('The target event already has a timetable'))
+
+        if self.old_event.duration > target_event.duration:
+            conflicts.append(_('The source event is too long'))
```

Comment:
Thinking about this again, maybe this is better?

```suggestion
            conflicts.append(_('The source event is longer than the target event'))
```